types/lodash/common/string.d.ts

```diff
@@ -690,21 +690,25 @@ declare module "../index" {
         template(
             string?: string,
             options?: TemplateOptions
-        ): TemplateExecutor;
+        ): TemplateExecutor<any>;
+        template<T>(
+            string?: string,
+            options?: TemplateOptions
+        ): TemplateExecutor<T>;
```

Comment:
Unfortunately, it's the [common mistake](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes) number 6.

Reply:
Ohh.. now I see it, thanks, I think I should close this PR then.